gfx/portraits/portrait_modifiers/50_beards_scripted_characters.txt

```diff
@@ -376,6 +376,11 @@ beards_scripted_characters = {
                 add = 200
 				is_godfrey_trigger = yes
             }
+            modifier = {
+                add = 225
+                age >= 50
```

Comment:
There may a problem of beardless Anduid since `is_anduin_wrynn_trigger` is still in `no_beard`.

Reply:
Anduin still appears beardless between ages 18-49 due to the added requirement of `age >= 50` and a higher weight value, so there doesn't seem to be a problem. But if you think this will cause problems, I can separate Anduin from the others and make it a new modifier with `age < 50` as a requirement?

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/230791929-5aac8413-aab3-4796-91ae-bb3e72910931.png)

</details>